iaso/serializers.py

```diff
@@ -16,26 +17,35 @@ def validate(self, attrs):
         """
         data = super().validate(attrs)
 
-        err_msg = "No active account found with the given credentials"
         request = self.context.get("request")
-        app_id = request.query_params.get("app_id", None)
-        if app_id and self.user.tenant_users.exists():
+        app_id = request.query_params.get("app_id")
```

Comment:
Shouldn't you use the `AppIdSerializer`?

```suggestion
        app_id = AppIdSerializer(data=request.query_params).get_app_id(raise_exception=False)
```

Reply:
lol, nice catch ;)